rector.php

```diff
@@ -25,10 +26,12 @@
     ->withSets([
         LevelSetList::UP_TO_PHP_83,
         SymfonySetList::SYMFONY_64,
+        SymfonySetList::SYMFONY_72,
         SymfonySetList::SYMFONY_CODE_QUALITY,
         SymfonySetList::SYMFONY_CONSTRUCTOR_INJECTION,
     ])
     ->withImportNames(true, true, false, true)
     ->withSkip([
         '**/config/bundles.php',
+        InlineClassRoutePrefixRector::class
```

Comment:
This rules remove the #[Route] annotation on controllers and it breaks the naming of the routes

Reply:
Do what I say not what I do,.. this could move into dedicated Mr. 

Rule is here https://getrector.com/rule-detail/inline-class-route-prefix-rector